tests/src/test/scala/whisk/core/entity/test/ExecManifestTests.scala

```diff
@@ -170,4 +176,22 @@ class ExecManifestTests
                     image.localImageName("r", "p", Some("tag")) shouldBe s"r/p/$name:tag"
             }
     }
+
+    it should "throw an error when configured manifest is invalid" in {
+        val config_manifest = "{\"nodejs\":[{\"kind\":\"nodejs\",\"image\":{\"name\":\"nodejsaction\"},\"deprecated\":true},{\"kind\":\"nodejs:6\",\"default\":true,\"image\":{\"name\":\"nodejs6action\"}}]}"
```

Comment:
A comment, that this JSON is supposed to be valid but missing a key would help readability grasping what's going on in that case. I'm fine with a string.